### PR TITLE
[#63] frostfs-adm: Fix contract wallet creation

### PR DESCRIPTION
**Pull request by dstepanov-yadro**
Create contract wallet only by init and update-config command.

### CODE REVIEW

**Comment by fyrchik** [comment]
@dstepanov-yadro we agreed to use `adm` prefix, without `frostfs` for new commit messages.

File: cmd/frostfs-adm/internal/modules/morph/group.go [hunk 1]
```diff
@@ -49,16 +49,15 @@ func initializeContractWallet(v *viper.Viper, walletDir string) (*wallet.Wallet,
...
-		if !os.IsNotExist(err) {
-			return nil, fmt.Errorf("can't open wallet: %w", err)
+		if os.IsNotExist(err) && createIfNotExists {
+			cmd.Printf("Contract group wallet is missing, initialize at %s\n", p)
+			return initializeContractWallet(v, walletDir)
 		}
-
-		cmd.Printf("Contract group wallet is missing, initialize at %s\n", p)
-		return initializeContractWallet(v, walletDir)
+		return nil, fmt.Errorf("can't open wallet: %w", err)
```
Comment (fyrchik) [inline cmd/frostfs-adm/internal/modules/morph/group.go:60]: `set-config` _should_ still work if contract wallet does not exist, it isn't used in any way in it.
Does it work after this change or am I missing something?

Comment (dstepanov-yadro) [reply cmd/frostfs-adm/internal/modules/morph/group.go:60]: fixed

File: cmd/frostfs-adm/internal/modules/morph/group.go [hunk 2]
```diff
@@ -53,12 +53,11 @@ func openContractWallet(v *viper.Viper, cmd *cobra.Command, walletDir string) (*
...
-		if !os.IsNotExist(err) {
-			return nil, fmt.Errorf("can't open wallet: %w", err)
+		if os.IsNotExist(err) {
+			cmd.Printf("Contract group wallet is missing, initialize at %s\n", p)
+			return initializeContractWallet(v, walletDir)
 		}
-
-		cmd.Printf("Contract group wallet is missing, initialize at %s\n", p)
-		return initializeContractWallet(v, walletDir)
+		return nil, fmt.Errorf("can't open wallet: %w", err)
```
Comment (fyrchik) [inline cmd/frostfs-adm/internal/modules/morph/group.go:60]: Can we remove these lines from the diff, please? Seem identical.

**Review by fyrchik — APPROVED** [review]

File: cmd/frostfs-adm/internal/modules/morph/group.go (hunk 2 above)
Comment (dstepanov-yadro) [reply cmd/frostfs-adm/internal/modules/morph/group.go]: fixed

**Review by fyrchik — APPROVED** [review]

**Pull request merged**